src/Tasks/Backup/BackupJob.php

```diff
@@ -302,7 +314,8 @@ protected function copyToBackupDestinations(string $path): void
             ->each(function (BackupDestination $backupDestination) use ($path) {
                 try {
                     if (! $backupDestination->isReachable()) {
-                        throw new Exception("Could not connect to disk {$backupDestination->diskName()} because: {$backupDestination->connectionError()}");
+                        $sanitizedError = CredentialSanitizer::sanitizeMessage($backupDestination->connectionError());
```

Comment:
I'm ok with throwing an exception here. You don't need to sanitize the message.

Reply:
Do refactor the exception to how it's done for the other exceptions. See `BackupFailed`